src/libraries/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -21,27 +21,27 @@
     <Compile Include="System\Security\AccessControl\FileSystemAuditRule.cs" />
     <Compile Include="System\Security\AccessControl\FileSystemRights.cs" />
     <Compile Include="System\Security\AccessControl\FileSystemSecurity.cs" />
-    <Compile Include="$(CoreLibSharedDir)Interop\Windows\Interop.BOOL.cs" Link="Common\Interop\Windows\Interop.BOOL.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Interop.BOOL.cs" Link="Common\Interop\Windows\Interop.BOOL.cs" />
```

Comment:
There are several files that use `Link="..."` instead of `<Link>...</Link>`. I am thinking about unifying everything on `<Link>...</Link>`.

Reply:
Begin able to specify additional metadata as xml attributes is a newer feature and available only in SDK style projects.

It is also the default that VS generates (when adding a file as a link via the UI) so we should probably normalize on that.

Reply:
If we were to use the Link attribute for the Common Interop files, it would produce very long lines. 180 characters long line would not be unusual. I believe that reasonable max line length is around 120.

The runtime libraries .csproj files are optimized for readability and hand-editing. They cannot be edited most of the time via drag&drop in VS solution explorer anyway. I do not see value in aligning to what VS generates.

I have opened #1266 to discuss this further.